coverage_assessment.R

```diff
@@ -41,7 +41,118 @@ nut <- get_data()
 
 # Problem 3: Prevalence of acute undernutrition/malnutrition ----
 
+library(dplyr)
+
+## Load functions in R directory ----
+for (f in list.files("R", full.names = TRUE)) source (f)
+
+
+# Read data ----
+nut <- get_data()
+
+
+####MAM
+
+with(nut, sum(muac >= 11.5 & muac <12.5  & age >=6 & age <=59 & oedema == 2, na.rm= TRUE))
+
+##number of moderate acute malnourished (MAM) cases children 6-59 months old 
+
+Male_MAM <- with(nut, sum(muac >= 11.5 & muac <12.5 & age >=6 & age <=59 & oedema == 2 & sex == 1 , na.rm= TRUE ))
+
+##assigning moderate acute malnourished (MAM) male 6-59 months old to Male_MAM
+
+Female_MAM <- with(nut, sum(muac >= 11.5 & muac <12.5 & age >=6 & age <=59 & oedema == 2 & sex == 2 , na.rm= TRUE ))
+
+##assigning  moderate acute malnourished (MAM) female 6-59 months old to Female_MAM
+
+UM_MAM <- with(nut, sum(muac >= 11.5 & muac <12.5 & age >=6 & age <=59 & oedema == 2 & county == "Urban Montserrado" , na.rm= TRUE))
+
+##assigning moderate acute malnourished (MAM) cases children 6-59 months old  in Urban Montserrado to UM_MAM
+
+GB_MAM <- with(nut, sum(muac >= 11.5 & muac <12.5 &age >=6 & age <=59 & oedema == 2 & county != "Urban Montserrado" , na.rm= TRUE))
+
+##assigning  moderate acute malnourished (MAM) cases children 6-59 months old other than Urban Montserrado to GB_MAM
+
+MAM <- data.frame(Male_MAM, Female_MAM, UM_MAM, GB_MAM)
+
+## creating a data frame for MAM
+
+if(interactive()) View(MAM)
+
+##Viewing MAM data
+
+den1 <- with(nut, sum(age >=5 & age < 60, na.rm = TRUE ))
```

Comment:
Here your age cut-offs is not consistent with what you are doing above. Important to be consistent given that this will give you your denominator for your prevalence estimate. It should be: `age >= 6 & age < 60`

Reply:
Yeah, just realised and made the changes.. Thanks!